### PR TITLE
Fix two bugs in replicatedservice.py

### LINKED ISSUE #45: namespace belongs in metadata in Kubernetes resources
It's in the wrong place in types/replicatedservice/v1/replicatedservice.py

### LINKED ISSUE #46: Label key "name" is an anti-pattern
Because users confuse it with metadata.name. Please use app: instead.

https://github.com/kubernetes/deployment-manager/blob/master/types/replicatedservice/v1/replicatedservice.py#L102

### PR DESCRIPTION
**Pull request by jackgr**
Fixes #45. Fixes #46.

### CODE REVIEW

File: types/replicatedservice/v1/replicatedservice.py [hunk 1]
```diff
@@ -41,9 +41,9 @@ def GenerateConfig(context):
       'properties': {
           'apiVersion': 'v1',
           'kind': 'Service',
-          'namespace': namespace,
           'metadata': {
               'name': service_name,
+              'namespace': namespace,
```
Comment (vaikas-google) [inline types/replicatedservice/v1/replicatedservice.py]: Are you sure the namespace moves here? As I read the api, it's part of the path for creating the resource and not part of metadata?

Comment (jackgr) [inline types/replicatedservice/v1/replicatedservice.py]: Yes. The api paths include the namespace, but in the configuration, the namespace lives in the metadata section. See, for example https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/skydns-svc.yaml.in.

Comment (vaikas-google) [inline types/replicatedservice/v1/replicatedservice.py]: Ack. That part was confusing me.

**Comment by vaikas-google** [comment]
LGTM

**Pull request merged**